content/docs/contributing/to-docs/style-guide/guidance.md

```diff
@@ -2,59 +2,80 @@
+- **-ed words**: Be cautious with words that end in *-ed*. Words that end in *-ed* can also be ambiguous. Use determiner phrases such as *that is* to clarify the usage of words that end in *-ed*. Take the word "based", for example.
 
-    Type | Definition | Example
-    :--| :-- | :--
-    Idiom | A phrase established to have a meaning that is not discernible from the individual words. | Forward+ rendering provides *the best of both worlds*.
-    Slang | Informal and nonstandard vocabulary. | *Chill* for a bit, while the O3DE project compiles.
-    Colloquialism | Ordinary and familiar conversational words and phrases, especially those that might be specific to a region. | ... On Create will execute the function *ASAP*.
-    Jargon | Specialized terms used in a particular field that are difficult for others to understand. | ... entering the *vertical-slice* phase of development.
+    | Do | Don't |
+    | :--| :-- |
+    | Atom is a renderer *that is* based on ... | Atom is a renderer based on ... |
 
-* Use optional words and phrases to clarify, such as *the*, *that*, *a*, *an*, *because*, *after*, *although*, and *might*.
-* Use commas to make sentences easier to read and comprehend. Use the serial or "Oxford" comma in lists. For example:
+- **Helpful words**: Use optional words and phrases to clarify, such as *the*, *that*, *a*, *an*, *because*, *after*, *although*, and *might*.
 
-    "**Asset Processor** checks for new files, detects changed files, and uses asset manifest files to process game-ready assets."
+- **Commas**: Use commas to make sentences easier to read and comprehend. Use the serial or "Oxford" comma in lists. In the following example, observe that a comma appears between the last item in the list and the preceding item.
 
-* Avoid Latin phrases such as *etc.*, *vs.*, *i.e.*, and *e.g.*.
+    **Example**: "**Asset Processor** checks for new files, detects changed files, and uses asset manifest files to process game-ready assets."
+
+
+
+## Voice and tone
+
+Use [active voice](https://writing.wisc.edu/handbook/style/ccs_activevoice/) and present-tense verbs wherever possible. Speak simply, respectfully, and professionally.
+
+Refer to the user as "you"; don't use "we".
+
+## Acronyms, abbreviations, and Latin phrases
+
+When you use acronyms, introduce them in their expanded form followed by the acronym in parentheses. Subsequent on-page references can be by acronym only.
+
+Don't abbreviate common words or Latin phrases; use the complete word or a similar one.
+
+**Exceptions**
+Acronyms that don't need to be spelled out:
+- Common file formats (Examples: JSON, PDF, JPEG, PNG)
+- Other common technical terms (Examples: URL, ID)
+
+| Do | Don't |
+| :-- | :-- |
+| Welcome to Open 3D Engine (O3DE)! | Welcome to O3DE! |
+| For example, Example: | e.g., Ex: |
+| versus, compared to | vs. |
+
+
+## Idioms, slang, colloquialisms, jokes, or jargon
```

Comment:
Meant to add jokes in the section text. 

Good point! There is some humor that's appropriate for user guides. The intent here is specifically to prevent jokes that may be inappropriate or don't translate well, as jokes have a tendency to be regional. 

I'm fine with removing jokes as a way to resolve this. But I'm also interested in building a proper case for how jokes can be addressed in docs, if at all. 

Tagging @sptramer, as he's had notable experiences with jokes in docs.

Reply:
Thanks for the insight. I've updated the rule on jokes to be a suggestion to avoid. I think this is sufficient for now.